Assets/Scripts/Model/Graph.cs

```diff
@@ -21,5 +21,35 @@ internal void OnDeserializedMethod(StreamingContext context)
       if (edges == null)
         edges = new List<Edge>();
     }
+
+    public IList<Edge> FindEdgesByNodes(Node node1, Node node2)
+    {
+      var argSet = new HashSet<Node> {node1, node2};
+      return (
+        from edge in edges
+        where argSet.Contains(edge.nodeFrom) && argSet.Contains(edge.nodeTo)
+        select edge
```

Comment:
This seems overly complicated and slow. Why not use a simple equality comparison?